apps/admin-gui/src/app/vos/components/application-detail/application-detail.component.html

```diff
@@ -68,7 +68,7 @@ <h2 class="page-subtitle">
       </ng-container>
       <ng-container matColumnDef="value">
         <th mat-header-cell *matHeaderCellDef></th>
-        <td mat-cell *matCellDef="let userData"> {{userData.value}} </td>
+        <td mat-cell *matCellDef="let userData" style="word-break: break-all"> {{userData.value}} </td>
```

Comment:
Please, try to avoid inlining style. You can create some CSS class in the `styles.scss` which then can be reused across the whole application. You can name it something like `break-all-words`.